backend/src/Designer/Services/Implementation/SchemaModelService.cs

```diff
@@ -285,8 +286,9 @@ private Json.Schema.JsonSchema GenerateJsonSchemaFromXsd(Stream xsdStream)
 
         private async Task UpdateCSharpClasses(AltinnAppGitRepository altinnAppGitRepository, ModelMetadata modelMetadata, string schemaName)
         {
-            string classes = _modelMetadataToCsharpConverter.CreateModelFromMetadata(modelMetadata);
-            await altinnAppGitRepository.SaveCSharpClasses(classes, schemaName);
+            string csharpClasses = _modelMetadataToCsharpConverter.CreateModelFromMetadata(modelMetadata);
+            Compiler.CompileToAssembly(csharpClasses);
```

Comment:
Compilation can be added inside the converter in JsonMetadataToCsharpConverter.CreateModelFromMetadata.